src/forms/modelForm.tsx

```diff
@@ -189,163 +178,194 @@ class ModelForm extends React.Component<ModelFormProps, ModelFormState> {
+                itemRenderer={renderStringItem}
+                itemPredicate={filterString}
+                noResults={<MenuItem disabled={true} text="No results." />}
+              />
+            </FormGroup>
+
+            <FormGroup label="Model Number*" inline={false}>
+              <Field
+                placeholder="model_number"
+                onChange={this.handleChange}
+                value={values.model_number}
+                field="model_number"
+              />
+            </FormGroup>
+            <FormGroup label="Display Color">
+              <Field
+                field="display_color"
+                type="color"
+                value={values.display_color ? values.display_color : "#394B59"}
+                onChange={this.handleChange}
+              />
+            </FormGroup>
+          </Card>
+          <Card>
+            <RadioGroup
+              label="Mount Type*"
+              className="radio-group"
+              onChange={(e: any) => {
                 this.setState({
-                  values: updateObject(values, { vendor: vendor }),
+                  values: updateObject(values, {
+                    model_type: e.currentTarget.value,
+                  }),
                 });
               }}
-              itemRenderer={renderStringItem}
-              itemPredicate={filterString}
-              noResults={<MenuItem disabled={true} text="No results." />}
-            />
-          </FormGroup>
-
-          <FormGroup label="Model Number (required)" inline={false}>
-            <Field
-              placeholder="model_number"
-              onChange={this.handleChange}
-              value={values.model_number}
-              field="model_number"
-            />
-          </FormGroup>
-          <FormGroup label="Height (required)" inline={false}>
-            <Field
-              field="height"
-              placeholder="height"
-              value={values.height}
-              onChange={this.handleChange}
-            />
-          </FormGroup>
-          <FormGroup label="Display Color">
-            <Field
-              field="display_color"
-              type="color"
-              value={values.display_color ? values.display_color : "#394B59"}
-              onChange={this.handleChange}
-            />
-          </FormGroup>
-          <FormGroup label="Number of Network Ports " inline={false}>
-            <Field
-              field="num_network_ports"
-              value={values.num_network_ports}
-              onChange={this.handleChange}
-            />
+              selectedValue={values.model_type}
+            >
+              <Radio
+                label={MountTypes.RACKMOUNT}
+                value={MountTypes.RACKMOUNT}
+              />
+              <Radio
+                label={MountTypes.BLADE_CHASSIS}
+                value={MountTypes.BLADE_CHASSIS}
+              />
+              <Radio label={MountTypes.BLADE} value={MountTypes.BLADE} />
```

Comment:
My new least favorite thing the linter has done

Reply:
hehehehehehehe